### PR TITLE
How to Secure Nginx with Let's Encrypt on Ubuntu 18.04

### PR DESCRIPTION
**Pull request by 1817666**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: MohammadHossein Fakhraei eng.fakhraei@gmail.com

### CODE REVIEW

**Comment by n-se** [comment]
Seems like you also pushed the CentOS tutorial into this branch...

File: tutorials/secure-nginx-with-let-s-encrypt-on-ubuntu-18-04/01.en.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+
+Certbot can automatically configure SSL for Nginx, but it needs to be able to find the correct `server` block in your config. It does this by looking for a `server_name `directive that matches the domain you're requesting a certificate for. If you're starting out with a fresh Nginx install, you can update the default config file:
+
+Create the directory for **example.com** as follows, using the `-p` flag to create any necessary parent directories:
+
+```bash
+mkdir -p /var/www/example.com/html
+```
+
+Next, assign ownership of the directory with the `$USER` environment variable:
```
Comment (n-se) [inline tutorials/secure-nginx-with-let-s-encrypt-on-ubuntu-18-04/01.en.md]: shouldn't it be owned by either `nginx` or `www-data` user?

File: tutorials/secure-nginx-with-let-s-encrypt-on-ubuntu-18-04/01.en.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+```
+
+The certbot Let's Encrypt client is now installed and ready to use.
+
+## Step 4 - Obtaining a Certificate
+
+Certbot provides a variety of ways to obtain SSL certificates, through various plugins. The Nginx plugin will take care of reconfiguring Nginx and reloading the config whenever necessary:
+
+```bash
+certbot --nginx -d example.com -d www.example.com
```
Comment (n-se) [inline tutorials/secure-nginx-with-let-s-encrypt-on-ubuntu-18-04/01.en.md]: Maybe also `--rsa-key-size 4096` for a more secure cert as on the other tutorial

File: tutorials/secure-nginx-with-let-s-encrypt-on-ubuntu-18-04/01.en.md [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+(c) The contribution was provided directly to me by some other person
+    who certified (a), (b) or (c) and I have not modified it.
+
+(d) I understand and agree that this project and the contribution are
+    public and that a record of the contribution (including all personal
+    information I submit with it, including my sign-off) is maintained
+    indefinitely and may be redistributed consistent with this project
+    or the license(s) involved.
+
+Signed-off-by: [MohammadHossein Fakhraei eng.fakhraei@gmail.com]
```
Comment (n-se) [inline tutorials/secure-nginx-with-let-s-encrypt-on-ubuntu-18-04/01.en.md]: As noted on the other tutorial `Firstname Lastname <your@email.com>` would be the correct form of a sign off

Comment (1817666) [reply tutorials/secure-nginx-with-let-s-encrypt-on-ubuntu-18-04/01.en.md]: Done.

File: tutorials/secure-nginx-with-let-s-encrypt-on-ubuntu-18-04/01.en.md (hunk 2 above)
Comment (1817666) [reply tutorials/secure-nginx-with-let-s-encrypt-on-ubuntu-18-04/01.en.md]: Edited.

File: tutorials/secure-nginx-with-let-s-encrypt-on-ubuntu-18-04/01.en.md (hunk 1 above)
Comment (1817666) [reply tutorials/secure-nginx-with-let-s-encrypt-on-ubuntu-18-04/01.en.md]: I test with `$USER` (root) and everything was OK. I think in Ubuntu (or maybe Debian Base) we don't need set ownership to `nginx `or `www-data`. To be sure, I need check it in Debian too.
What's your opinion?

Comment (n-se) [reply tutorials/secure-nginx-with-let-s-encrypt-on-ubuntu-18-04/01.en.md]: We can keep it, yes. Seems like it doesnt matter who owns it.

**Comment by n-se** [comment]
Ok, looks ready. But you need to remove the second tutorial in this merge request which is for CentOS. You have two files here... :)

**Comment by 1817666** [comment]
> Ok, looks ready. But you need to remove the second tutorial in this merge request which is for CentOS. You have two files here... :)

Done.

**Pull request merged**